src/index.html

```diff
@@ -7,12 +7,71 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+      class="product-card"
+      data-qa="card"
+    >
+      <img
+        class="product-card__image"
+        src="images/imac.jpeg"
+        alt="Product logo"
+      />
+
+      <div class="product-card__name">
+        APPLE A1419 iMac 27" Retina
+        <br />
+        5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="product-card__code">Product code: 195434</div>
+
+      <div class="product-card__info">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="product-card__review">Reviews: 5</p>
+      </div>
+
+      <div class="product-card__price">
+        <p class="product-card__price-title">Price:</p>
+
+        <p class="product-card__price-value">$2,199</p>
+      </div>
+
+      <a
+        href="#buy"
+        class="product-card__buy"
+        data-qa="hover"
```

Comment:
The 'href' attribute in the link should lead to the product page or be a placeholder that indicates it is a link to the product purchase page, like 'href="#"'. The current value '#buy' suggests an anchor link on the same page, which might not be the intended behavior.